system/services/presideObjects/SqlSchemaSynchronizer.cfc

```diff
@@ -56,6 +56,11 @@ component {
 				tableVersionExists = StructKeyExists( versions, "table" ) and StructKeyExists( versions.table, obj.meta.tableName );
 				tableExists        = tableVersionExists or _getTableInfo( tableName=obj.meta.tableName, dsn=obj.meta.dsn ).recordCount;
 
+				if( StructKeyExists( obj.meta, "dbsync" ) and not obj.meta.dbsync ) {
```

Comment:
Would prefer:

```cfc
if ( IsBoolean( obj.meta.dbsync ?: "" ) ) && !obj.meta.dbsync ) {
}
```

Given this is used throughout, could even add a helper function to make more readable:

```cfc
if ( _skipSync( obj.meta ) ) { // ...
```
```cfc
private boolean function _skipSync( required struct meta ) {
    return IsBoolean( arguments.meta.dbsync ?: "" ) && !arguments.meta.dbsync;
}
```

Reply:
Hi @DominicWatson ,

I've done this changes please check latest [commit](https://github.com/pixl8/Preside-CMS/pull/900/commits/1a93a4da1c22efcbdb8ffddb69297e14478effc9)